pkg/awsapi/generate/add_import.go

```diff
@@ -0,0 +1,76 @@
+//go:build ignore
```

Comment:
Could we contribute a change to ifacemaker instead of maintaining this?

Reply:
There's an open [PR](https://github.com/vburenin/ifacemaker/pull/47) that claims to fix this issue in ifacemaker, but it has not been reviewed in months. I agree that it'd be best to have it fixed upstream.

Reply:
Have you tried the change from the PR by any chance? I'm curious if its works

Reply:
Added a comment to the PR :smile:

Reply:
>Have you tried the change from the PR by any chance? I'm curious if its works

That PR's fix does not work. I tried it on both https://github.com/vburenin/ifacemaker/issues/12 and our `generate-aws-interfaces` script. 

(I had actually also tested that potential fix before but it wasn't working then either, otherwise my plan was to use that user's fork).

Reply:
Now we have write access shall we fix ifacemaker instead of maintaining this in our own repo?

Reply:
Hmm, that fix might take some time. The program I wrote can't be used as-is in ifacemaker. The implementation in ifacemaker will also be more complex because a more complete solution would not use a dot import but rather prefix the non-local packages with the remote package's name. I can open an issue in our repo to track this though. WDYT?

Reply:
@Himangini - what are your thought on doing this as a follow-up? My concern if workarounds like this is that they have a habit of becoming the long-term solution.

Reply:
>My concern with workarounds like this is that they have a habit of becoming the long-term solution.

Agreed. Best would be to prioritize work on the follow-up issue so it doesn't become a long-term solution.

Reply:
I agree with both 👍🏻  @cPu1 can you create a follow-up issue and add it here as a comment, please. I'll bring this in refinement/planning next week.

Reply:
Awesome thanks @cPu1  and @Himangini

Reply:
I have opened https://github.com/weaveworks/eksctl/issues/4994